content/practices/user-research-introduction-session/index.md

```diff
@@ -0,0 +1,127 @@
+when:
+- Given the importance of research early in the project, this session is typically done as soon as possible after Kickoff in advance of any stakeholder interviews or exploratory user research.
+what:
+- "Whiteboard or digital version like [Miro](https://miro.com/)"
+- Dry erase markers
+
+remote: false
+miro_template_url: "" 
+---
+## How to Use this Method
+
+### Sample Agenda & Prompts
+1. Write the agenda on the whiteboard or shared digital workspace:
+   
+   - Types (10 min)
+   - Roles (10 min)
+   - Flow (10 min)
+   - Dos & Don’ts (10 min)
+   - Role Play (25 min)
+
+1. First, describe the three main **types** of research by drawing/showing a three part Venn diagram, complete with a squiggly line representing time beneath it, and explaining how they relate to one another.
+   
+   - Exploratory — what are the problems?
+   - Validation — is this solution valuable?
+   - Usability — is this solution easy to use?
+
+   ![Research Workshop Types](/images/practices/user-research-introduction-session/research-types.jpg)
+
+1. Next, describe the main **roles** involved in research, writing each on the board:
+   
+   Critical:
+  
+   - Lead Interviewer — facilitates interview
+   - Note-taker(s) — ideally 1 primary & 1 secondary to help ensure good coverage
+   
+   Optional:
+   
+   - Co-Pilot — supports Lead Interviewer
+   - Media Wrangler — handles recording
+   - Transcriber — types up interview as it unfolds
+
+1. With roles covered, describe the general flow of research by breaking it into discrete steps:
+   
+   Recruit & Schedule
+   - Pick days for research
+   - Reach out to users
+   - Schedule slots
+   
+   Prep
+   - Goals
+   - Assumptions
+   - Questions
+   - Design artifacts (if necessary)
+   
+   Conduct Interviews
+   - Facilitate!
+   - Take notes in an appropriate manner (e.g. sticky notes, transcription, etc.) in voice of user, capturing sentiment using smiley faces
+   - Record aspects of the session that matter (e.g. audio, video, etc.)
+   
+   Synthesize Findings
+   - All types: group like comments across users to find trends
+   - Usability only: create pass/fail table to quickly see which aspects of the design worked and which did not
+   - See Research Synthesis for more information
+   
+   {{< callout >}}
+   Tip: Drawing these four steps on the board showing a progression can be really helpful to visualize how they flow together
+   {{< /callout >}}
+   
+1. Share **biases** to be aware of and key **“dos”** and **“don’ts”** when conducting research by printing out or projecting this **Research Dos & Don'ts Guide** 
+
+   ***EDITOR NOTE: The above Research Dos & Don'ts Guide is an internal google doc. Please change, publish the doc, or find an alternative***
```

Comment:
@fanibahar Is there a different resource we can link to here, or can we convert the internal doc to a companion blog post, PDF, etc.?

Reply:
I added it as a page. See update.